data.sql

```diff
@@ -423,4 +423,34 @@ VALUES
 	( (SELECT id from animals where name = 'Blossom'), (SELECT id from vets where name = 'Stephanie Mendez'), '2020-05-24' ),
 	( (SELECT id from animals where name = 'Blossom'), (SELECT id from vets where name = 'William Tatcher'), '2021-01-11' );
 
-  COMMIT;
\ No newline at end of file
+  COMMIT;
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (name, age, date_of_graduation) VALUES('Maisy Smith', 26, '2000-01-17');
+INSERT INTO vets (name, age, date_of_graduation) VALUES('Stephanie Mendez', 64, '2000-05-04');
```

Comment:
- [ ] Vet `Stephanie Mendez` graduation date should be `1981-05-17` instead of `2000-05-04`. We can work on this as well.